packages/puppeteer/test/integration/doc-dylang.test.ts

```diff
@@ -27,7 +27,7 @@ describe('doc-dylang.html', function () {
   before(async function () {
     // const app: express.Application = express()
     const app: express.Application = express();
-    app.use(express.static(path.resolve(__dirname,  '..', 'fixtures')));
+    app.use(express.static(path.resolve(__dirname, '..', 'fixtures')));
```

Comment:
The formatting changes to the existing tests were recommended by `npm run fmt` when I ran it as part of validating the new test